addons/google_calendar/utils/google_calendar.py

```diff
@@ -28,6 +28,22 @@ class GoogleCalendarService():
     def __init__(self, google_service):
         self.google_service = google_service
 
+    @requires_auth_token
+    def get_event(self, eventId, sync_token, token=None, timeout=TIMEOUT):
```

Comment:
Isn't it possible to improve get_events and/or extract common code to avoid code duplication ?